### PR TITLE
Oct dev update

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: content/posts/irods-development-update-october-2024.md [hunk 1]
```diff
@@ -0,0 +1,288 @@
+Title: iRODS Development Update: October 2024
+Date: 2024-10-15 12:00
```
Comment (trel) [inline content/posts/irods-development-update-october-2024.md:2]: 16 now, sorry.

File: content/posts/irods-development-update-october-2024.md [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+Author: Kory Draughn
+Slug: irods-development-update-october-2024
+Status: published
+
+
+Hello reader,
+
+The temperature is starting to decrease. Hopefully, everyone is staying warm. Let's get into the development update.
```
Comment (trel) [inline content/posts/irods-development-update-october-2024.md:10]: northern hemisphere, yes.

File: content/posts/irods-development-update-october-2024.md [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+
+For the iRODS HTTP API, work to introduce user mapping plugins has been merged. The next release will allow administrators to modify the user mapping in real time. Not only that, but work to add support for local JWT validation has begun. This will help organizations using identity providers which do not expose an introspection endpoint for token validation.
+
+To wrap up, iRODS S3 API 0.3.0 is available. This release includes support for **ListParts** and **ListMultipartUploads** operations and more importantly, the performance improvements for multipart upload. To learn more, see the release notes at [https://github.com/irods/irods_client_s3_api/releases/tag/0.3.0](https://github.com/irods/irods_client_s3_api/releases/tag/0.3.0).
```
Comment (trel) [inline content/posts/irods-development-update-october-2024.md:24]:
```suggestion
To wrap up, iRODS S3 API 0.3.0 is available. This release includes support for **ListParts** and **ListMultipartUploads** operations and, more importantly, a ~40% performance improvement for multipart upload. To learn more, see the release notes at [https://github.com/irods/irods_client_s3_api/releases/tag/0.3.0](https://github.com/irods/irods_client_s3_api/releases/tag/0.3.0).
```

**Review by trel — COMMENTED** [review]
looking good.

File: content/posts/irods-development-update-october-2024.md (hunk 1 above)
Comment (korydraughn) [reply content/posts/irods-development-update-october-2024.md]: Done

File: content/posts/irods-development-update-october-2024.md (hunk 2 above)
Comment (korydraughn) [reply content/posts/irods-development-update-october-2024.md]: Replaced with wording about SC24.

Please review.

File: content/posts/irods-development-update-october-2024.md (hunk 3 above)
Comment (korydraughn) [reply content/posts/irods-development-update-october-2024.md]: Done

**Comment by korydraughn** [comment]
Ready for final review.

File: content/posts/irods-development-update-october-2024.md [hunk 4]
```diff
@@ -0,0 +1,288 @@
+Title: iRODS Development Update: October 2024
+Date: 2024-10-16 10:00
+Author: Kory Draughn
+Slug: irods-development-update-october-2024
+Status: published
+
+
+Hello reader,
+
+Supercomputing 2024 is here again! If you're attending, swing by the RENCI booth and say hi. Let's get into the development update.
```
Comment (trel) [inline content/posts/irods-development-update-october-2024.md:10]:
```suggestion
[Supercomputing 2024]({filename}/pages/sc24.html) is in Atlanta soon! If you're attending, swing by the RENCI booth and say hi. Let's get into the development update.
```

Comment (korydraughn) [reply content/posts/irods-development-update-october-2024.md]: Done

**Comment by trel** [comment]
the time of the squashing has come.

**Comment by korydraughn** [comment]
Squashed

**Pull request merged**